src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,32 @@
+# Download and execute Chocolatey install script
+Set-ExecutionPolicy Bypass -Scope Process -Force
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
+iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
```

Comment:
Is there a reason you used single quotes here or will this work?
```suggestion
iex ((New-Object System.Net.WebClient).DownloadString("https://community.chocolatey.org/install.ps1"))
```

Reply:
@dav3r resolved with [85cbaed](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/85cbaed820d6399f84419b360ec22677e0369050)

Reply:
@dav3r resolved with [e815c96](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/e815c96f405a6a1c545504e1cd420e8e9d411303)